web_frontend/osmaxx/job_progress/middleware.py

```diff
@@ -29,11 +32,41 @@ def update_order_if_stale(extraction_order):
     get_cached_or_set(
         'extraction_order_{}_progress'.format(extraction_order.id),
         update_order, extraction_order,
-        timeout=timedelta(minutes=1).seconds
+        on_cache_hit=_log_cache_hit,
+        timeout=timedelta(minutes=1).seconds,
     )
 
 
 def update_order(extraction_order):
+    _log_cache_miss(extraction_order)
     conversion_client = get_authenticated_api_client()
     conversion_client.update_order_status(extraction_order)
+    if logger.isEnabledFor(logging.DEBUG):
+        message = "Fetched, updated and cached ExtractionOrder {extraction_order} progress: {progress}".format(
+            extraction_order=extraction_order.id,
+            progress=ExtractionOrderState.label(extraction_order.state),
+        )
+        logger.debug(message)
     return extraction_order.state
+
+
+def _log_cache_miss(extraction_order):
+    if not logger.isEnabledFor(logging.INFO):
+        return
+    message = "Cache miss - Re-fetching progress of pending ExtractionOrder {extraction_order}.".format(
+        extraction_order=extraction_order.id,
+    )
+    logger.info(message)
```

Comment:
Is `info` the right level? `debug` seems more appropriate for me here, because the only time I want these messages to _pollute_ my console is during bug-hunting. Is there a special reason for the info level?